### PR TITLE
Keep postgres capture blacklist in redis

### PR DESCRIPTION
**Pull request by macobo**
- Remove dead code around caching
- Keep blacklist of postgres capture in redis

    The list is refetched every 1-2 minutes from redis by API servers.
    
    To add to list, use redis-cli and run the following:
    
    ```
    LPUSH postgres_capture_blacklist 536 572 700
    ```
    
    Replacing the numbers with appropriate team_ids. 

    Already populated the blacklist on production

## Checklist

- [ ] All querysets/queries filter by Organization, Team, and User (if this PR affects ANY querysets/queries).
- [ ] Django backend tests (if this PR affects the backend).
- [ ] Cypress end-to-end tests (if this PR affects the frontend).

### CODE REVIEW

File: posthog/api/test/base.py [hunk 1]
```diff
@@ -21,7 +20,6 @@ def _create_user(self, email: str, password: Optional[str] = None, first_name: s
 
     def setUp(self):
         super().setUp()  # type: ignore
-        clear_cache()
```
Comment (mariusandra) [inline posthog/api/test/base.py:24]: This made sense for clickhouse tests when we were storing persistent information about captured unique elements inside redis. It seems like that's no longer being used, so probably fine to remove.

Comment (macobo) [reply posthog/api/test/base.py:24]: Yup, all dead code. :+1:

File: posthog/redis.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+            server = fakeredis.FakeServer()
+            _client = fakeredis.FakeStrictRedis(server=server)
+            _client._server = server
+        elif settings.REDIS_URL:
+            _client = redis.from_url(settings.REDIS_URL, db=0)
+    return _client
+
+
+class GoldenRetriever:
```
Comment (fuziontech) [inline posthog/redis.py:30]: most adorable class name ever
https://i.gifer.com/BEhj.gif

**Comment by fuziontech** [comment]
I was thinking about this more. Could we just add a field to team that is 'blacklisted' and use that?

**Comment by macobo** [comment]
Fair point, let's do that

**Pull request closed without merging**